contracts-periphery/src/test/BatchSendwToll.t.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "./UmbraBatchSend.t.sol";
+
+contract BatchSendwTollTest is UmbraBatchSendTest {
+
+    address mockOwner;
+
+    function setUp() public override {
+
+        vm.label(alice, "Alice");
+        vm.label(bob, "Bob");
+        vm.label(address(this), "UmbraBatchSendTest");
+
+        umbraPrevBal = token.balanceOf(umbra);
+        
+        // mockOwner = UmbraToll(umbra).owner();
+        // emit log_named_address("Umbra owner addr is", mockOwner);
+        // vm.deal(mockOwner, type(uint256).max);
+        // vm.startPrank(mockOwner);
+        // UmbraToll(umbra).setToll(10);
+        // vm.stopPrank();
+        toll = 0;
+
+        tip(dai, address(this), type(uint256).max);
+        vm.deal(address(this), type(uint256).max);
```

Comment:
If we're inheriting from the other test contract, we can call `super.setUp()` instead of duplicating the logic.

Also, I'd recommend setting balances significantly smaller than max uint256, like `100 ether` is probably plenty. Max uint is not necessary/realistic, and likely to result in overflow if you ever try transferring funds to that address